docker/start_workspace_tmux.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
```

Comment:
Let's shift this to `tools/` --- I'm trying to keep `docker/` for the critical scripts needed for CI.